### ISSUE TITLE
infinite loading; persists between Operating Systems as long as I'm logged in 

### ISSUE THREAD

**Issue by RandomPrimary2004** [darkreader/darkreader#9804]
# Bug Report

## Bug Description
When on Firefox, clicking the Darkreader icon next to the search bar leads to a nonstop instance of "loading, please wait". This persists between different Firefox installations between systems, always when logged in. Logging out does not absolve the affected browser of this issue, but uninstalling and deleting the cache and config for Firefox, then proceeding to reinstall and install Dark Reader from the add-ons tab without logging in seems to work. This issue is also internet-wide, not limited to a single site.   

## Steps To Reproduce
Install Firefox on Linux or Windows 10. Log in. Let Firefox Sync download Dark Reader automatically for you. Click the Dark Reader icon in an attempt to adjust settings. 

## Expected Behavior
Settings menu for dark reader appears inside the window produced by clicking the icon. 

## Actual Behavior
Settings window loads, but the settings themselves do not. Text reading "Loading, please wait" appears, and does not go away regardless of time waited. I've waited for a full hour while away from my computer and it still did not load the settings. 

## Screenshots
![Screenshot from 2022-09-11 00-35-23](https://user-images.githubusercontent.com/111301912/189514043-3930da6a-95bf-495c-9346-e918bb9206db.png)

## System Information
OS: LMDE 5 (elsie) x86_64 
Kernel: 5.10.0-18-amd64 
Uptime: 1 hour, 2 mins 
Packages: 2937 (dpkg), 15 (flatpak) 
Shell: bash 5.1.4 
Resolution: 1280x1024, 1360x768 
WM: awesome 
Theme: Mint-Y [GTK3] 
Icons: Mint-Y [GTK3] 
Terminal: gnome-terminal 
CPU: Intel i7-2600 (8) @ 3.800GHz 
GPU: NVIDIA GeForce GTX 650 
Memory: 2936MiB / 11955MiB 

- OS: LMDE 5 elsie kernel 5.10.0-18-amd64, Microsoft Windows 10 21H2
- Browser: Firefox 104.0.2
- Dark Reader Version: 4.9.57

## Additional Context
Does not affect a fresh Firefox installation unless using Sync, does not affect Chromium installed on the same computer.

**Comment by Gusted** [comment]
Hi @RandomPrimary2004,

Could you try installing Dark Reader without using the Firefox Sync feature? It's possible Dark Reader tried to load corrupted data from the sync.

**Comment by RandomPrimary2004** [comment]
Yes, without logging in, it seems to work. in fact, I logged out of my browser, and upon checking the extension just now, it works flawlessly. I've logged in while the extension is active, and that's fixed it for good, I assume because it synced the "good" working data to my account, instead of loading the "bad" data from it. I will leave the issue open and let someone who knows better than me close it if needed, because I feel like this bug could be either from Firefox its self, or from some misconfiguration on Dark Reader's side.
